moler/connection.py

```diff
@@ -146,6 +146,23 @@ def send(self, data, timeout=30, encrypt=False):
         encoded_data = self.encode(data)
         self.how2send(encoded_data)
 
+    def change_newline_seq(self, newline_seq="\n"):
+        """
+        Method to change newline char(s). Useful when connect from one point to another if newline chars change (i.e. "\n", "\r\n")
+        :param newline_seq: Sequence of chars to send as new line char(s)
+        :return: Nothing
+        """
+        stro = ""
+        strn = ""
```

Comment:
don't use tricky names. Why not just `newline_old`, `newline_new`? At first glimpse I thought `stro` is string output :-)